lib/simp/beaker_helpers.rb

```diff
@@ -2,7 +2,7 @@ module Simp; end
 
 module Simp::BeakerHelpers
   require 'simp/beaker_helpers/version'
-  DEFAULT_PUPPET_AGENT_VERSION = '1.8.3'
+  DEFAULT_PUPPET_AGENT_VERSION = '1.10.2'
```

Comment:
Default should be PUPPET_VERSION or the version of the puppet gem I have bundled.